package.json

```diff
@@ -38,7 +38,7 @@
     "adobe"
   ],
   "scripts": {
-    "test": "nyc --reporter=html --reporter=text mocha test --timeout 60000 --recursive && npm run checkout-node-fetch && npm run test-node-fetch",
+    "test": "nyc --reporter=html --reporter=text mocha test --timeout 20000 --recursive && npm run checkout-node-fetch && npm run test-node-fetch",
```

Comment:
why are you decreasing the timeout if tests were failing due to timeouts?

Reply:
@jdelbick I tried both, in the retro the notes say 20000. Either value above and including 600000 failed as timeouts

Reply:
No need to reduce the value if it timeouts :)
20000ms is a start value, you can increase over that. 
More interesting would be to understand why it takes so long.